modules/concept-docs/pages/durability-replication-failure-considerations.adoc

```diff
@@ -16,7 +16,14 @@ include::{version-server}@sdk:shared:partial$durability-replication-failure-cons
 include::{version-server}@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=syncrep]
 include::{version-server}@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=syncrep3]
 
-include::{version-server}@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=older]
+== Older Server Versions
+// include::{version-server}@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=older]
+If a version of Couchbase Server lower than 6.5 is being used then the fallback is 'client verified' durability.
+
+WARN: Client Verified durability is supported in xref:3.2@durability-replication-failure-considerations.adoc#older-server-versions[Node.js SDK 3.2] but not in 4.0.
```

Comment:
Shouldn't this be WARNING ?
https://docs.antora.org/antora/latest/asciidoc/admonitions/